.github/workflows/tests.yml

```diff
@@ -1,18 +1,23 @@
 ---
-name: Test collection
+name: CI
 
 concurrency:
-  group: ${{ github.head_ref }}
+  group: ${{ github.head_ref || github.run_id }}
   cancel-in-progress: true
 
 on:  # yamllint disable-line rule:truthy
   pull_request:
     branches: [main]
   workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * *"
 
 jobs:
+  ansible-lint:
+    uses: ansible-network/github_actions/.github/workflows/ansible-lint.yml@main
```

Comment:
This is the older lint, we should use the fresh lint reference.

Reply:
Updating to newer lint would cause CI to break and I do want really want to force merge PRs with CI failures. These changes are meant to be a part of the last known 2.9 compatible release which cannot be one that passes the latest ansible-lint, since it enforce collections to require an ansible-core version that's not EoLed.